portal-web/docroot/html/taglib/ui/form_navigator/lexicon/sections.jspf

```diff
@@ -113,27 +113,26 @@
 				</c:otherwise>
 			</c:choose>
 
-			Liferay.once(
-				'<portlet:namespace />formReady',
-				function(event) {
-					if (!sectionContent.hasClass('in')) {
-						if (focusField.length) {
+			if (focusField.length) {
```

Comment:
We are passing ```focusField``` into ```Liferay.Util.focusFormField``` so if there is nothing in that array, then we should not be focusing on anything.

We want to leverage the ```formReady``` event in all cases, but it is the ```sectionContent``` panel opening and closing that we just need to check.